obspy/core/tests/test_response.py

```diff
@@ -298,6 +298,42 @@ def test_response_with_no_units_in_stage_1(self):
             out, [0 + 9869.2911771081963j, 0 + 19738.582354216393j,
                   0 + 39477.164708432785j])
 
+    def test_paz_zero_values(self):
```

Comment:
Can you please rename your test cases to something like `test_resp_from_paz_settings_zeros()` and similar more descriptive names?